application.yml

```diff
@@ -0,0 +1,10 @@
+artemis:
+  athena:
+    enabled: false
```

Comment:
this files is placed next to the packaged JAR and overrides the default config from application.yml-files within the packaged JAR